src/Sylius/Behat/Context/Setup/CustomerContext.php

```diff
@@ -114,6 +114,16 @@ public function theStoreHasEnabledCustomerAccountWithPassword($email, $password)
     }
 
     /**
+     * @Given there is a customer account :name with email :email identified by :password
```

Comment:
It would be better to change this step and declare explicit first name and surname. Otherwise, we could not be sure, that there is a space in `$name` variable, so `$names[1]` call could cause fatal error.
`there is a customer account :firstName :surName with email :email identified by :password`